libs/ml/src/utilities/word2vec_utilities.cpp

```diff
@@ -0,0 +1,45 @@
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "ml/utilities/word2vec_utilities.hpp"
+
+namespace fetch {
+namespace ml {
+namespace utilities {
+
+// Timestamp for logging
+std::string GetStrTimestamp()
```

Comment:
This should be declared in the header file

Reply:
It can't be, because it is not templated so it wouldn't pass static analysis.